ambry-clustermap/src/test/java/com.github.ambry.clustermap/HelixParticipantTest.java

```diff
@@ -160,6 +167,95 @@ public void testSetReplicaSealedState() throws IOException {
     listIsExpectedSize(sealedReplicas, 0, listName);
   }
 
+  /**
+   * Tests setReplicaStoppedState method for {@link HelixParticipant}
+   * @throws IOException
+   */
+  @Test
+  public void testSetReplicaStoppedState() throws IOException {
+    //setup HelixParticipant and dependencies
+    String partitionId1 = "partitionId1";
+    String partitionId2 = "partitionId2";
+    ReplicaId replicaId1 = createMockAmbryReplica(partitionId1);
+    ReplicaId replicaId2 = createMockAmbryReplica(partitionId2);
+    String hostname = "localhost";
+    int port = 2200;
+    String instanceName = ClusterMapUtils.getInstanceName(hostname, port);
+    String instanceNameDummy = ClusterMapUtils.getInstanceName("dummyHost", 2200);
+    HelixParticipant helixParticipant =
+        new HelixParticipant(new ClusterMapConfig(new VerifiableProperties(props)), helixManagerFactory);
+    HelixParticipant helixParticipantDummy =
+        new HelixParticipant(new ClusterMapConfig(new VerifiableProperties(propsDummy)), helixManagerFactory);
+    helixParticipant.participate(Collections.EMPTY_LIST);
+    helixParticipantDummy.participate(Collections.EMPTY_LIST);
+    HelixManager helixManager = helixManagerFactory.getZKHelixManager(null, null, null, null);
+    HelixAdmin helixAdmin = helixManager.getClusterManagmentTool();
+    InstanceConfig instanceConfig = new InstanceConfig("testInstanceId");
+    helixAdmin.setInstanceConfig(clusterName, instanceName, instanceConfig);
+    helixAdmin.setInstanceConfig(clusterName, instanceNameDummy, null);
+
+    //Make sure the current stoppedReplicas list is null
+    List<String> stoppedReplicas = ClusterMapUtils.getSealedReplicas(instanceConfig);
+    assertNull("stoppedReplicas is not null", stoppedReplicas);
+
+    String listName = "stoppedReplicas list";
+
+    //Check that invoking setReplicaStoppedState with a non-AmbryReplica ReplicaId throws an IllegalArgumentException
+    ReplicaId nonAmbryReplica = createMockNotAmbryReplica(partitionId1);
+    try {
+      helixParticipant.setReplicaStoppedState(nonAmbryReplica, true);
+      fail("Expected an IllegalArgumentException here");
+    } catch (Exception e) {
```

Comment:
`catch (IllegalArgumentException e)` (don't need the `assertTrue()`)

Reply:
fixed